Fix `log.debug` `TypeError` in `conda.core.package_cache_data` (#11679)
conda/conda: news/11679-fix-typeerror.md

### Enhancements

* <news item>

### Bug fixes

> * Fixed TypeError encountered when log is debug and package JSON cannot be read (#11679)

Reviewer: ```suggestion
* Fixed `TypeError` encountered when logging is set to DEBUG and the package's JSON cannot be read. (#11679)
```
